sdk/storage/azure-storage-blob/azure/storage/blob/_shared/request_handlers.py

```diff
@@ -145,3 +149,121 @@ def add_metadata_headers(metadata=None):
         for key, value in metadata.items():
             headers['x-ms-meta-{}'.format(key.strip())] = value.strip() if value else value
     return headers
+
+
+def serialize_batch_body(requests, batch_id):
```

Comment:
This is great, this would actually enable us to use the generated `batch_submit()` method.